Dockerfile-devcore-centos-9

```diff
@@ -0,0 +1,47 @@
+# vi:ft=dockerfile
+FROM rockylinux:9
+
+# SYSTEM
+RUN yum clean all
+RUN yum upgrade -y
+RUN yum install -y wget which
+RUN yum install -y sudo
+
+# ENVIRONMENT
+RUN yum install -y git perl ruby
+RUN yum install -y perl-Data-Dumper perl-IPC-Cmd
+RUN yum install -y gcc gcc-c++ make ant ant-junit maven
+RUN yum install -y rpm-build createrepo zlib* rsync
+# Install OpenJDK 8 and 11
+RUN yum install -y java-1.8.0-openjdk-devel java-11-openjdk-devel
+
+# Set up alternatives for Java 8
+RUN update-alternatives --install /usr/bin/java java /usr/lib/jvm/java-1.8.0-openjdk/bin/java 1
+RUN update-alternatives --set java /usr/lib/jvm/java-1.8.0-openjdk/bin/java
+
+# Set up alternatives for Java 8
+RUN update-alternatives --install /usr/bin/javac javac /usr/lib/jvm/java-1.8.0-openjdk/bin/javac 1
+RUN update-alternatives --set javac /usr/lib/jvm/java-1.8.0-openjdk/bin/javac
```

Comment:
# Set up alternatives for Java 8
RUN update-alternatives --install /usr/bin/java java /usr/lib/jvm/java-1.8.0-openjdk/bin/java 1
RUN update-alternatives --set java /usr/lib/jvm/java-1.8.0-openjdk/bin/java
RUN update-alternatives --install /usr/bin/javac javac /usr/lib/jvm/java-1.8.0-openjdk/bin/javac 1
RUN update-alternatives --set javac /usr/lib/jvm/java-1.8.0-openjdk/bin/javac